Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 group :development do
-  gem 'verify-metadata-generator', :git => 'git@github.com:alphagov/verify-metadata-generator'
```

Comment:
Any reason we're switching to https protocol rather than git?

Reply:
we don't need to use git protocol to fetch the repo - it's not going to be edited by downloading this way

Reply:
I guess it's a bit more convenient for third party users who don't have ssh keys set up, now I think about it.